src/safe_apps/tests/test_views.py

```diff
@@ -29,6 +30,10 @@ def test_json_payload_format(self) -> None:
                 "description": safe_app.description,
                 "chainIds": safe_app.chain_ids,
                 "provider": None,
+                "accessControl": {
+                    "type": SafeApp.AccessControlPolicy.NO_RESTRICTIONS,
```

Comment:
Instead of using `SafeApp.AccessControlPolicy.NO_RESTRICTIONS`, use the expected `str` on the payload side. This way, we can catch if the API was broken due to a refactor (eg.: if we rename `SafeApp.AccessControlPolicy.NO_RESTRICTIONS` everywhere then the test would be successful even though it might not be the result that we wanted)